scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt

```diff
@@ -828,6 +828,30 @@ class RegexPatternValidationCheckTest {
     assertThat(outContent.toString().trim()).isEqualTo(REGEX_CHECK_PASSED_OUTPUT_INDICATOR)
   }
 
+  @Test
+  fun testFileContent_translatableString_inPrimaryStringsFile_fileContentIsCorrect() {
+    val prohibitedContent = "<string name=\"test\">Translatable</string>"
+    tempFolder.newFolder("testfiles", "app", "src", "main", "res", "values")
+    val stringFilePath = "app/src/main/res/values/strings.xml"
+    tempFolder.newFile("testfiles/$stringFilePath").writeText(prohibitedContent)
+
+    runScript()
+
+    assertThat(outContent.toString().trim()).isEqualTo(REGEX_CHECK_PASSED_OUTPUT_INDICATOR)
+  }
+
+  @Test
+  fun testFileContent_translatableString_inTranslatedPrimaryStringsFile_fileContentIsCorrect() {
+    val prohibitedContent = "<string name=\"test\">Translatable</string>"
+    tempFolder.newFolder("testfiles", "app", "src", "main", "res", "values-ar")
```

Comment:
can we have a test case for 

`app/src/main/res/values-ar/<str>/strings.xml` too

Reply:
Is this actually allowed by Android resources? I was under the impression that value files should generally live directly under their respective values folders?

Reply:
Hmm https://stackoverflow.com/questions/4930398/can-the-android-layout-folder-contain-subfolders lays out contradictory information. I'm actually thinking that maybe we need a check to prohibit subdirectories instead of supporting subdirectories in these types of patterns.

Reply:
![image](https://user-images.githubusercontent.com/54740946/133330954-01bcc3b6-9a91-4824-8f8b-af09b6f7cd3e.png)
This is the new android studio template for new projects 
I guess it is allowed

Reply:
@FareesHussain is that representing directory structure? That doesn't seem right at face value since themes.xml (night) means values-night/themes.xml. What's the directory structure for that folder ('project view' not 'Android view')?

Reply:
Ah my bad, it is values-night/themes.xml

Reply:
New check added.